src/WebJobs.Script/Diagnostics/DiagnosticEventConstants.cs

```diff
@@ -25,5 +25,8 @@ internal static class DiagnosticEventConstants
 
         public const string LinuxConsumptionTimeZoneErrorCode = "AZFD0010";
         public const string LinuxConsumptionTimeZoneErrorHelpLink = "https://go.microsoft.com/fwlink/?linkid=2250165";
+
+        public const string MissingFunctionsWorkerRuntimeErrorCode = "AZFD0011";
+        public const string MissingFunctionsWorkerRuntimeHelpLink = "https://go.microsoft.com/fwlink/?linkid=2257963";
```

Comment:
Do we have anything tracking the configuration of the forward link?

Reply:
I'm going to jump over and add text for this right now (that PR may be out before this goes in). Will link them together.

Reply:
FYI -- PR for this is here -- https://github.com/MicrosoftDocs/azure-docs/pull/119164